token-metadata/program/src/instruction/burn.rs

```diff
@@ -103,7 +104,75 @@ pub fn burn_nft(
 #[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
 pub enum BurnArgs {
     V1 {
-        /// Required authorization data to validate the request.
-        authorization_data: Option<AuthorizationData>,
+        /// The amount of the token to burn
+        amount: u64,
     },
 }
+
+/// Burn an asset.
+///
+/// # Accounts:
+///
+///
+///   0.   `[signer, writable]` Owner or Utility Delegate of the asset
+///   1.   `[optional, writable]` Collection Metadata account
+///   2.   `[writable]` Metadata account
+///   3.   `[optional, writable]` Edition account
+///   4.   `[writable]` Mint account
+///   5.   `[writable]` Token account
+///   6.   `[optional, writable]` Parent Edition account
+///   7.   `[optional]` Parent Mint account
+///   8.   `[optional]` Parent Token account
+///   9.   `[optional, writable]` Edition Marker account
+///  10.   `[optional, writable]` Token record account
+///  11.   `[]` System program
+///  12.   `[]` Instruction sysvar account
+///  13.   `[]` SPL Token Program
```

Comment:
Nitpick: I feel like these descriptions should match the shank annotations, but maybe I'm being too picky?